cmd/appgw-ingress/main.go

```diff
@@ -96,11 +97,12 @@ func main() {
 
 	apiConfig := getKubeClientConfig()
 	kubeClient := kubernetes.NewForConfigOrDie(apiConfig)
+	istioCrdClient := istio.NewForConfigOrDie(apiConfig)
```

Comment:
can order the clients instantiation in order of kubeClient, crdClient, istioCrdClient